### PR TITLE
reference: add hibernate region config

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 
Added hibernate region config for TiKV as an Experimental feature

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1669
### Which version does your change affect? 

TiDB 3.0/3.1/Dev

### CODE REVIEW

**Comment by lilin90** [comment]
@kissmydb FYI. The hibernate Region description has been added, but it's still an experimental feature currently.

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 1]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Regions 中 Leader 和 Follower 之间的心跳间隔。
```
Comment (lilin90) [inline dev/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Region 中 Leader 和 Follower 之间的心跳间隔。
```

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 2]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
 + 默认值：3s
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```
Comment (lilin90) [inline dev/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 3]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
...
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
```
Comment (lilin90) [inline dev/reference/configuration/tikv-server/configuration-file.md]: One more question: do we need to add `(**实验特性**)` for this item?

File: v3.0/reference/configuration/tikv-server/configuration-file.md [hunk 4]
```diff
@@ -316,6 +316,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Regions 中 Leader 和 Follower 之间的心跳间隔。
```
Comment (dcalvin) [inline v3.0/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Region 中 Leader 和 Follower 之间的心跳间隔。该配置项仅在静默 Region 打开后才可生效。
```

File: v3.1/reference/configuration/tikv-server/configuration-file.md [hunk 5]
```diff
@@ -316,6 +316,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Regions 中 Leader 和 Follower 之间的心跳间隔。
```
Comment (dcalvin) [inline v3.1/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Region 中 Leader 和 Follower 之间的心跳间隔。该配置项仅在静默 Region 打开后才可生效。
```

File: v3.0/reference/configuration/tikv-server/configuration-file.md [hunk 6]
```diff
@@ -316,6 +316,16 @@ raftstore 相关的配置项。
 + 默认值：3s
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```
Comment (dcalvin) [inline v3.0/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```

File: v3.1/reference/configuration/tikv-server/configuration-file.md [hunk 7]
```diff
@@ -316,6 +316,16 @@ raftstore 相关的配置项。
 + 默认值：3s
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```
Comment (dcalvin) [inline v3.1/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
```

**Comment by lilin90** [comment]
@BusyJay PTAL~ Thanks!

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 8]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Region 中 Leader 和 Follower 之间的心跳间隔。该配置项仅在静默 Region 打开后才可生效。
```
Comment (BusyJay) [inline dev/reference/configuration/tikv-server/configuration-file.md]: Not just for hibernate region, but also work for active regions.

Comment (dcalvin) [reply dev/reference/configuration/tikv-server/configuration-file.md]: 修改对 Region 的状态检查间隔时间。 Is this enough?

Comment (BusyJay) [reply dev/reference/configuration/tikv-server/configuration-file.md]: LGTM

File: v3.0/reference/configuration/tikv-server/configuration-file.md [hunk 9]
```diff
@@ -316,6 +316,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对静默 Region 的状态检查间隔时间，同时该值也决定了静默 Region 中 Leader 和 Follower 之间的心跳间隔。该配置项仅在静默 Region 打开后才可生效。
```
Comment (lilin90) [inline v3.0/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 修改对静默 Region 的状态检查间隔时间。
```

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 10]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
...
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
+
+### `raftstore.peer-stale-state-check-interval`
+
++ 修改对 Region 的状态检查间隔时间
```
Comment (lilin90) [inline dev/reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 修改对 Region 的状态检查间隔时间。
```

**Comment by dcalvin** [comment]
PTAL again @lilin90 Thanks.

File: dev/reference/configuration/tikv-server/configuration-file.md [hunk 11]
```diff
@@ -315,6 +315,16 @@ raftstore 相关的配置项。
 + 默认值：3s
 + 最小值：0
 
+### `raftstore.hibernate-regions` (**实验特性**)
+
++ 打开或关闭静默 Region。打开后，如果 Region 长时间处于非活跃状态，即被自动设置为静默状态。静默状态的 Region 可以降低 Leader 和 Follower 之间心跳信息的系统开销。可以通过 `raftstore.peer-stale-state-check-interval` 调整 Leader 和 Follower 之间的心跳间隔。
++ 默认值：false
```
Comment (BusyJay) [inline dev/reference/configuration/tikv-server/configuration-file.md]: 这是指 master 么？master 默认值为 true。

Comment (dcalvin) [reply dev/reference/configuration/tikv-server/configuration-file.md]: master branch? 应该是吧？

**Pull request merged**